cairo/src/word.rs

```diff
@@ -0,0 +1,429 @@
+//! The Cairo language works natively for field elements in the finite field with
+//! modulus 0x800000000000011000000000000000000000000000000000000000000000001
+//! This is the hexadecimal value for 2 ^ 251 + 17 * 2 ^ 192 + 1
+//! Our Pallas curves have 256 bits, so Cairo native instructions will fit.
+
+use crate::definitions::*;
+use crate::helper::CairoFieldHelpers;
+use ark_ff::FftField;
+use o1_utils::field_helpers::FieldHelpers;
+
+/// A Cairo instruction for simulation
```

Comment:
if this is an instruction, why is it called a word?

Reply:
I guess this comes from the previous version of the code where I had different entities for CairoWord and CairoInstruction. But I switched to only having CairoWords. Basically, all cairo instructions are cairo words, but not all cairo words are instructions (where instructions stands for 64-bit words with the flag and offsets structure). I will unify notation so it's clearer.